pkg/apis/resources/v1alpha1/types.go

```diff
@@ -108,6 +108,10 @@ const (
 	// SeccompProfileSkip is a constant for a label on a Pod which indicates that this Pod should not be considered for
 	// defaulting of its seccomp profile.
 	SeccompProfileSkip = "seccompprofile.resources.gardener.cloud/skip"
+
+	// PodTopologySpreadConstraintsSkip is a constant for a label on a Pod which indicates that this Pod should not be considered for
+	// adding the pod-template-hash selector to the topology spread constraint.
+	PodTopologySpreadConstraintsSkip = "topologyspreadconstraints.resources.gardener.cloud/skip"
```

Comment:
Alternative naming would be also

```suggestion
	PodTopologySpreadConstraintsSkip = "topology-spread-constraints.resources.gardener.cloud/skip"
```

to make it consistent with `projected-token-mount.resources.gardener.cloud/skip` and `token-invalidator.resources.gardener.cloud/skip`.